components/inputs/input-date-range.js

```diff
@@ -145,6 +145,21 @@ class InputDateRange extends FormElementMixin(RtlMixin(LocalizeCoreElement(LitEl
 		`;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		changedProperties.forEach((oldVal, prop) => {
+			if (prop === 'startValue' || prop === 'endValue') {
+				this.setFormValue({
+					[`${this.name}-startValue`]: this.startValue,
+					[`${this.name}-endValue`]: this.endValue,
```

Comment:
I guess if they don't set a name, then they likely don't care about the form value anyway?